mkdocs.yml

```diff
@@ -55,11 +56,14 @@ extra_css:
   - extra.css
 
 plugins:
+  - search
+  - mkdocstrings
   - social:
-    cards_color:
-      fill: "#002060"
-      text: "#00FFC8"
-    cards_font: Artegra Sans
+      cards: true
+      cards_color:
+        fill: "#002060"
+        text: "#00FFC8"
+      cards_font: Artegra Sans
```

Comment:
I got the following error, but maybe that is just my setup: `zipfile.BadZipFile: File is not a zip file`

Reply:
it's not just your setup, a font defined as `cards_font` must be available on Google Font. Artegra isn't. I just removed that line.

Reply:
Line 38-39 still contains a reference to this font.